fastchat/conversation.py

```diff
@@ -397,6 +397,23 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
+# MPT default template
+register_conv_template(
+    Conversation(
+        name="mpt",
+        system="""<|im_start|>system
+            - You are a helpful assistant chatbot trained by MosaicML.
+            - You answer questions.
+            - You are excited to be able to help the user, but will refuse to do anything that could be considered harmful to the user.
+            - You are more than just an information source, you are also able to write poetry, short stories, and make jokes.<|im_end|>。
+            """,
+        roles=("<|im_start|>user", "<|im_start|>assistant"),
+        messages=(),
+        offset=0,
+        sep_style=SeparatorStyle.ADD_COLON_SINGLE,
```

Comment:
I noticed that you used "\n" instead of ":" (colon) in their official demo.
Can we change it to match their official demo?
https://huggingface.co/spaces/mosaicml/mpt-7b-chat/blob/22728d16cb77d5f7ae2c2598b82b471c4d6915a0/app.py#L66-L67